nucliadb_telemetry/nucliadb_telemetry/errors.py

```diff
@@ -101,3 +107,33 @@ def setup_error_handling(version: str) -> None:
             default_integrations=False,
         )
         sentry_sdk.set_tag("zone", settings.zone)
+
+
+class SentryHandler(EventHandler):
```

Comment:
If an `ImportError` occurs, we'll try to create a subclass of `None` and it will fail